corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -353,9 +357,23 @@ def build_query_prompts(self):
                     )
                     for i, validation in enumerate(prop.validations)
                 ]
+            if prop.group_key:
+                kwargs['group_key'] = prop.group_key
             prompts.append(QueryPrompt(**kwargs))
         return prompts
 
+    def build_query_prompt_groups(self):
+        prompts = []
+        prompt_group_properties = [prop for prop in self.module.search_config.properties if prop.is_group]
+        for prop in prompt_group_properties:
+            if prop.is_group:
```

Comment:
Isn't this always true based on `prompt_group_properties = [prop for prop in self.module.search_config.properties if prop.is_group]`?

Reply:
yes you're right [8e09ca8](https://github.com/dimagi/commcare-hq/pull/33828/commits/8e09ca81655f14b4d46f0474bc5c387ed79f5fd8)